Dockerfile

```diff
@@ -0,0 +1,43 @@
+#!/bin/echo docker build . -f
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: MPL-2.0
+#{
+# Copyright: 2021+ Philippe Coval <https://purl.org/rzr/>
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/ .
+#}
+
+FROM rust:bullseye
+LABEL maintainer="Philippe Coval (philippe.coval@astrolabe.coop)"
+
+RUN echo "#log: Setup system" \
+  && set -x \
+  && apt-get update \
+  && apt-get install -y \
+      --no-install-recommends \
+     ca-certificates \
+     libssl-dev \
+     pkg-config \
+  && apt-get upgrade -y \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/* \
+  && date -u
+
+ARG project=astarte-device-sdk-rust
+ARG workdir="/local/src/${project}"
+
+COPY . "${workdir}/"
+WORKDIR "${workdir}"
+RUN echo "#log: ${project}: Building sources" \
+  && set -x \
+  && export OPENSSL_NO_VENDOR=1 \
+  && cargo build  \
+  && cargo install --path . \
+  || cargo install --examples --path . \
```

Comment:
is this really needed?

Reply:
let me double check

Reply:
yes it's needed I have updated the commit message